src/app/theming/styles/schemas.scss

```diff
@@ -0,0 +1,188 @@
+@import '~igniteui-angular/lib/core/styles/themes/utilities';
+
+$black-color: #000000;
+$aqua-color: #00BFFF;
+
+$black-theme-palette: igx-palette($primary: $black-color, $secondary: $aqua-color);
```

Comment:
Ensure you set the $grays property to #fff for dark palettes.

Reply:
$grays added to the palette